Tests/LibM/test-math.cpp

```diff
@@ -267,3 +267,25 @@ TEST_CASE(acos)
     EXPECT_APPROXIMATE(acos(1), 0);
     EXPECT(isnan(acos(1.1)));
 }
+
+TEST_CASE(floor)
+{
+    EXPECT_EQ(floor(3.0 / 24.0), 0);
```

Comment:
Super nit, but `floor(0.125)` should be better here, as we don't have context here about the original failure coming from day calculation.